src/test/java/org/inferred/freebuilder/processor/AnalyserTest.java

```diff
@@ -460,6 +460,75 @@ public void twoGetters_interface() throws CannotGenerateCodeException {
     assertThat(builder.getGeneratorsByProperty().keySet()).containsExactly(name, age).inOrder();
   }
 
+  @Test
+  public void ignoredEqualsAndHashCode() throws CannotGenerateCodeException {
+    GeneratedBuilder builder = (GeneratedBuilder) analyser.analyse(model.newType(
+        "package com.example;",
+        "public class DataType {",
+        "  @org.inferred.freebuilder.IgnoredByEquals public abstract String getName();",
+        "  public static class Builder extends DataType_Builder {}",
+        "}"));
+
+    Property available = new Property.Builder()
```

Comment:
Did this get renamed at some point? I think it should be called "name", also below.

Reply:
Probably from my copy and pasting using your existing tests and forgot to have the variable name match the name in the property itself.